torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -270,6 +270,57 @@ RegisterOperators reg({
             return 0;
           };
         }),
+    Operator(
+        prim::ConstantChunk,
+        [](Node* node) {
+          int64_t chunks = node->i(attr::chunks);
+          int64_t dim = node->i(attr::dim);
+          return [=](Stack& stack) {
+            autograd::profiler::RecordFunction record("chunk");
+            at::Tensor t;
+            pop(stack, t);
+            auto result = at::chunk(t, chunks, dim);
+            stack.insert(stack.end(), std::make_move_iterator(result.begin()),
+                                      std::make_move_iterator(result.end()));
+            return 0;
+          };
+        }),
+    Operator(
+        prim::ListUnpack,
+        [](Node* node) -> Operation {
+          size_t num_outputs = node->outputs().size();
+          ListTypePtr lt = node->input()->type()->expect<ListType>();
+          if (lt->getElementType() == IntType::get()) {
+            return [=](Stack& stack) {
```

Comment:
Is there a way to template this?

```
Shared<ConstantList<T>> list;
pop(stack, list);
```

Reply:
Oh there surely is, but I asked you the same some time ago, and you said you preferred this style, so I just continued using it